docs/node/private-networks.mdx

```diff
@@ -0,0 +1,88 @@
+```
+or
+
+```bash
+kwild -r /path/to/root/dir --chain.p2p.private-mode --chain.p2p.whitelist-peers="b624b3395f22f8cb57cf86d58cbcd6b355d12e18,7b4b3b3b5f22f8cb57cf86d58cbcd6b355d12e18"
+```
+
+The current set of validators, persistent peers and seed nodes are automatically added to the whitelist. Therefore, the `whitelist_peers` field is used to restrict which non-validating read-only nodes can sync data from your node's P2P endpoint.
+
+The whitelist can also be updated dynamically using the `kwil-admin peers` commands. Refer to the [Kwil Admin Peers Subcommand](/docs/ref/kwil-admin/peers) documentation for more information.
+
+:::note
+In Private mode, to establish a connection with the other node, both nodes should add each other's node ID to the whitelist. If one of the nodes doesn't add the other node to it's whitelist, the connection will not be established, unless it is a seed node or a persistent peer or a current validator.
+:::
+
+## Onboarding a Read Node
+
+When onboarding a new read node to the network, the new node's `Node ID` should be added to the whitelist of at least one of the existing network nodes.
+Once the new node is added to an existing node's whitelist, the new node can use the existing node's `Node ID` as it's persistent peer or seed node.
+
+For example, to onboard a new node with the Node ID `new-node-id` to an existing node with the Node ID `existing-node-id`, follow the steps below:
+
+1. Add the new node to the whitelist of an existing node as shown below:
+    ```bash
+    kwil-admin peers add <new-node-id>
+    ```
+2. Verify that the new node is added to the whitelist of the existing node:
+    ```bash
+    kwil-admin peers list
+    ```
+3. The new node can join the network by connecting to the existing node as a seed node or persistent peer. As seeds nodes and persistent peers are automatically whitelisted, the new node should be able to connect to the existing node without explicitly adding the existing node ID to the whitelist.
+
+    ```toml
+    [chain.p2p]
+    seeds = "existing-node-id@localhost:26656"
+    ```
+    ```toml
+    [chain.p2p]
+    persistent_peers = "existing-node-id@localhost:26656"
+    ```
+## Onboarding a Validator Node
+
+When onboarding a new validator node to the network, the node should first be [added as a read node](#onboarding-a-read-node) to the network. Once the node is part of the network, the node can then [request the network to become a validator](/docs/node/tutorial#upgrading-the-second-node-to-a-validator). 
+
+Existing validators adds the joiner node to their whitelist when they approve the validator join request and if the request is rejected (expired), the joiner node will be removed from the whitelist of all the validators that approved the join request.
+If the validator join request is approved, the new validator node is automatically added to the whitelist of all the nodes in the network.
```

Comment:
Typo and clarity:

```suggestion
Existing validators add the joiner node to their whitelist when they approve the validator join request. If the request is rejected (expired), the joiner node will be removed from the whitelist of all the validators that previously approved the request.
If the validator join request reaches the 2/3 vote threshold, the new validator node is automatically added to the whitelist of all the nodes in the network.
```

Reply:
2/3rd threshold is when the validator gets approved, I can just rephrase it to say that explicilty